core/src/main/scala/cats/data/package.scala

```diff
@@ -43,6 +43,9 @@ package object data {
     def tell[L](l: L): Writer[L, Unit] = WriterT.tell(l)
   }
 
```

Comment:
Would you mind adding a doctest for the `IndexedStateFunctions.apply` here?

Reply:
Sure! Please see https://github.com/typelevel/cats/pull/2693/commits/b858c14a1b9e6f9a0fb95662a225928cb5a371e5